style.css

```
@@ -0,0 +1,227 @@
+@import url("https://fonts.googleapis.com/css?family=Poppins");
+body {
+  font-family: "Poppins";
+}
+header > div {
+  padding: 0 25px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px solid lightgray;
+}
+nav {
+  width: 600px;
+}
+nav ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+nav a {
+  text-decoration: none;
+  color: black;
+}
+.blackHeart {
+  width: 20px;
+  margin-right: 1rem;
+}
+.npm-logo {
+  width: 70px;
+}
+.search-img {
+  width: 20px;
+  display: flex;
+  align-items: center;
+}
+.slot-form-class {
+  background-color: rgba(0, 0, 0, 0.05);
+  
+  width: 1500px;
+  height: 25px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.slot-form-class span {
+  display: flex;
+  width: 100%;
+}
+
+.slot-form-class input {
+  background-color: rgba(0, 0, 0, 0.05);
+  border: none;
+  display: flex;
+  justify-content: flex-start;
+  height: 25px;
+  width: 100%;
+}
+.slot-form-class button {
+  border: none;
+  background-color: #fb3e44;
+  color: white;
+  display: flex;
+  align-self: flex-end;
+  height: 26px;
+}
+.slot-form-class label {
+  background-color: rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: flex-start;
+  height: 28px;
+  justify-items: flex-start;
+}
+.btn-join {
+  border-width: 1px;
+  border-style: solid;
+  border-color: black;
+  background-color: white;
+  color: black;
+  box-sizing: border-box;
+  padding: 5px 10px;
+  text-decoration: none;
+}
+.btn-log {
+  border-width: 1px;
+  background-color: white;
+  color: black;
+  box-sizing: border-box;
+  padding: 5px 10px;
+  text-decoration: none;
```

Puedes agregar un minimo para la propiedad width para que no se recorra tu texto
```suggestion
  text-decoration: none;
  min-width: 70px;
```